test/common/graph.js

```diff
@@ -55,81 +55,54 @@ describe('graph', function () {
             graph.print(writer);
             writer.closeFile();
             expect(writer.data.get('graph.mmd')).toEqual(`flowchart LR
-   \`org.acme.hr.base@1.0.0.State\`
    \`org.acme.hr.base@1.0.0.State\` --> \`concerto@1.0.0.Concept\`
-   \`org.acme.hr.base@1.0.0.TShirtSizeType\`
    \`org.acme.hr.base@1.0.0.TShirtSizeType\` --> \`concerto@1.0.0.Concept\`
-   \`org.acme.hr.base@1.0.0.EmployeeTShirtSizes\`
    \`org.acme.hr.base@1.0.0.EmployeeTShirtSizes\` --> \`org.acme.hr.base@1.0.0.SSN\`
    \`org.acme.hr.base@1.0.0.EmployeeTShirtSizes\` --> \`org.acme.hr.base@1.0.0.TShirtSizeType\`
-   \`org.acme.hr.base@1.0.0.Address\`
    \`org.acme.hr.base@1.0.0.Address\` --> \`concerto@1.0.0.Concept\`
    \`org.acme.hr.base@1.0.0.Address\` --> \`org.acme.hr.base@1.0.0.State\`
-   \`org.acme.hr.base@1.0.0.Time\`
```

Comment:
We appear to have lost orphaned nodes in the graph. We should still write these (before tree-shaking).